sql/parser/builtins.go

```diff
@@ -248,6 +248,33 @@ var builtins = map[string][]builtin{
 		return DString(strings.Replace(s, from, to, -1)), nil
 	}, DummyString)},
 
+	"translate": {stringBuiltin3(func(s, from, to string) (Datum, error) {
+		const deletionRune = rune(0)
```

Comment:
That's something I wanted to bring up actually. I wasn't sure if it would be possible for `'\x00'` to be a valid character even though it is ASCII null, given your input parsing. I'm wondering if passing the utf8 library's defined [`RuneError`](https://golang.org/src/unicode/utf8/utf8.go) might be safer for an in-band sentinel value for character removal, as it corresponds to an invalid unicode point. What do you think?